plugin/forward/setup_test.go

```diff
@@ -382,3 +382,45 @@ func TestNextAlternate(t *testing.T) {
 		}
 	}
 }
+
+func TestFailfastAllUnhealthyUpstreams(t *testing.T) {
+	tests := []struct {
+		input          string
+		shouldErr      bool
```

Comment:
this field looks redundant, it can be inferred as `expectedErr != ""`. Redundant data complicates code and may lead to ambiguities, inconsistencies and contradictions.

Reply:
Fixed.